File: dayTwo/python/level two/taskFive.py
number = int(input("Enter a number: "))
number_two = int(input("Enter a Second Number: "))
number_three = int(input("Enter a Third Number: "))
largest = number

if(number_two > largest):
    largest = number_two
    print(f"The largest is {number_two}")
if(number_three > largest):
    largest = number_three
    print(f"The largest is {number_three}")
else:
    print(f"The largest is {number}")

File: dayTwo/javascript/level two/taskfive.js
const prompt = require("prompt-sync")();

let firstNumber = Number(prompt("Enter a number: "))
let secondNumber = Number(prompt("Enter a number: "))
let thirdNumber = Number(prompt("Enter a number: "))

let largest = firstNumber

if(secondNumber > largest){
    largest = secondNumber
    console.log(`The largest is ${secondNumber}`)
}
else if(thirdNumber > largest){
    largest = thirdNumber
    console.log(`The largest is ${thirdNumber}`)
}
else{
    console.log(`The largest is ${firstNumber}`)
}
